### PR TITLE
Fix: pre-launch link check

### LINKED ISSUE #69: [Blocked by main site prod] Link updates & checks, Countly review, etc.
- [x] Switch env var to reference the main prod site instead of main staging site
- [x] Test for broken links between sites 
- [x] Change our readme to include links to main service - pr open #103 
- [x] Confirm Countly is successfully pulling in production data from both sites
- [x] If repo switch public, reconfigure repo perms to remove any unnecessary access

### PR DESCRIPTION
**Pull request by jessicaschilling**
This PR fixes a few broken or otherwise less-useful links in a final pre-launch check.

Ref #69. As the last item in the checklist, this also closes #69.

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/query.md`:
- The following link returned HTTP status code 302 (Found):
  https://web3.storage/files
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check the retu...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ... to see the values for some of the more commonly-used attributes returned by...
                                             ^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...f pins.  If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
### `docs/how-tos/retrieve.md`:
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...is guide, we'll use the gateway at dweb.link, but you can see more worldwide...
                                             ^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...uy7pcu, you can make a URL for the dweb.link gateway as follows: bafybeidd2g...
                                             ^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... so if you're looking to link to one of your own files, you don't even have ...
                                             ^^^^^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ... you have a client, you can call client.get, passing in a CID string:  The g...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...S data: files and unixFsIterator(). The files method returns an array of Web...
                                             ^^^^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- Possibly missing comma found.
  ```
  ...ent to <FILE NAME>. Desktop/output-file.txt Your complete command should loo...
                                             ^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ... can also use the client library to get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```
### `docs/how-tos/store.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...s such as fetch  (opens new window) and File  (opens new window). This guide...
                                             ^^^^
  ```
- Possibly missing comma found.
  ```
  ....storage package to your dependencies:  First we need to create a Web3.Stora...
                                             ^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”.
  ```
  ...ent method: If you expect to be loading a lot of large files, you may be bet...
                                             ^^^^^^^^
  ```
- Apostrophe might be missing. Did you mean “file’s” (curly possessive apostrophe) or “file's” (straight possessive apostrophe)? (Some terms such as “farmers’ market” have a possessive apostrophe after the first noun.)
  ```
  ...one content archive, and linking to the files inside is much easier if each ...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...an array of Files, uploading is simple: IMPORTANT Deleting files from the We...
                                             ^^^^^^^^^
  ```
- Possibly missing article found.
  ```
  ... IMPORTANT Deleting files from the Web3.Storage site's Files page  (opens ne...
                                             ^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ...to guide on retrieval. You can also get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```

File: docs/how-tos/retrieve.md [hunk 1]
```diff
@@ -164,13 +164,13 @@ You can also use the client library to get more information about the status of
...
-[quickstart-guide]: ../quickstart/README.md
+[quickstart-guide]: ../README.md#quickstart
 [concepts-content-addressing]: ../concepts/content-addressing.md
 [howto-store]: ./store.md
 [howto-query]: ./query.md
 
-[reference-js-web3response-files]: ../reference/client-library.md#return-value-2
-[reference-js-web3response-unixfsiterator]: ../reference/client-library.md#return-value-2
+[reference-js-web3response-files]: ../reference/client-library.md#retrieve-files
+[reference-js-web3response-unixfsiterator]: ../reference/client-library.md#retrieve-files
```
Comment (terichadbourne) [inline docs/how-tos/retrieve.md:173]: @zebateira flagging you re the linking to a tab question

Comment (terichadbourne) [reply docs/how-tos/retrieve.md:173]: @jessicaschilling It took two of us looking carefully at this to figure out what makes sense here. The paragraph the link is in is talking not about the `get` method that you use to retrieve files (in which case I'd agree with you here), but about the `files` method that is something returned by the get method, which is why the more specific return value section makes more sense. The Unix link isn't being used, so I'll go in and make a commit to have a single link to the response section.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/query.md`:
- The following link returned HTTP status code 302 (Found):
  https://web3.storage/files
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check the retu...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ... to see the values for some of the more commonly-used attributes returned by...
                                             ^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...f pins.  If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
### `docs/how-tos/retrieve.md`:
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...is guide, we'll use the gateway at dweb.link, but you can see more worldwide...
                                             ^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...uy7pcu, you can make a URL for the dweb.link gateway as follows: bafybeidd2g...
                                             ^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... so if you're looking to link to one of your own files, you don't even have ...
                                             ^^^^^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ... you have a client, you can call client.get, passing in a CID string:  The g...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...S data: files and unixFsIterator(). The files method returns an array of Web...
                                             ^^^^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- Possibly missing comma found.
  ```
  ...ent to <FILE NAME>. Desktop/output-file.txt Your complete command should loo...
                                             ^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ... can also use the client library to get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```
### `docs/how-tos/store.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...s such as fetch  (opens new window) and File  (opens new window). This guide...
                                             ^^^^
  ```
- Possibly missing comma found.
  ```
  ....storage package to your dependencies:  First we need to create a Web3.Stora...
                                             ^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”.
  ```
  ...ent method: If you expect to be loading a lot of large files, you may be bet...
                                             ^^^^^^^^
  ```
- Apostrophe might be missing. Did you mean “file’s” (curly possessive apostrophe) or “file's” (straight possessive apostrophe)? (Some terms such as “farmers’ market” have a possessive apostrophe after the first noun.)
  ```
  ...one content archive, and linking to the files inside is much easier if each ...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...an array of Files, uploading is simple: IMPORTANT Deleting files from the We...
                                             ^^^^^^^^^
  ```
- Possibly missing article found.
  ```
  ... IMPORTANT Deleting files from the Web3.Storage site's Files page  (opens ne...
                                             ^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ...to guide on retrieval. You can also get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```

File: docs/how-tos/retrieve.md (hunk 1 above)
Comment (jessicaschilling) [reply docs/how-tos/retrieve.md]: Thanks!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/how-tos/query.md`:
- The following link returned HTTP status code 302 (Found):
  https://web3.storage/files
- Possibly missing preposition found.
  ```
  ... files. When you store a file with Web3.Storage, you receive a content ident...
                                             ^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...re's how to include it in your project: IMPORTANT Remember to check the retu...
                                             ^^^^^^^^^
  ```
- Possibly missing preposition found.
  ```
  ... is valid and has been uploaded to Web3.Storage, the status method will retu...
                                             ^^^^^^^
  ```
- Possible agreement error.
  ```
  ...an the total size of the files on disk. pins contains an array of objects de...
                                             ^^^^^^^^^^^^^
  ```
- Possible agreement error.
  ```
  ... fast retrieval using the IPFS network. deals contains an array of objects d...
                                             ^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ... to see the values for some of the more commonly-used attributes returned by...
                                             ^^^^^^^^^^^^^
  ```
- Possibly missing comma found.
  ```
  ...f pins.  If you haven't yet explored in depth how to store data using Web3.S...
                                             ^^^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
### `docs/how-tos/retrieve.md`:
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...is guide, we'll use the gateway at dweb.link, but you can see more worldwide...
                                             ^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ...uy7pcu, you can make a URL for the dweb.link gateway as follows: bafybeidd2g...
                                             ^^^^
  ```
- Use ‘own’ only if you want to emphasize that something belongs to a person or thing or was done by a person or thing. Consider writing “your”.
  ```
  ... so if you're looking to link to one of your own files, you don't even have ...
                                             ^^^^^^^^
  ```
- If a new sentence starts here, add a space and start with an uppercase letter.
  ```
  ... you have a client, you can call client.get, passing in a CID string:  The g...
                                             ^^^
  ```
- An apostrophe may be missing.
  ```
  ...S data: files and unixFsIterator(). The files method returns an array of Web...
                                             ^^^^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- In American English, the usual prepositions after ‘different’ are “from” or “than”.
  ```
  ...save the file to. This can be different to <FILE NAME>. Desktop/output-file....
                                             ^^
  ```
- Possibly missing comma found.
  ```
  ...ent to <FILE NAME>. Desktop/output-file.txt Your complete command should loo...
                                             ^^^
  ```
- The verb “check out” can sound colloquial in this context. Consider using a more formal alternative.
  ```
  ...h how to store data using Web3.Storage, check out the storage how-to guide f...
                                             ^^^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ... can also use the client library to get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```
### `docs/how-tos/store.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...s such as fetch  (opens new window) and File  (opens new window). This guide...
                                             ^^^^
  ```
- Possibly missing comma found.
  ```
  ....storage package to your dependencies:  First we need to create a Web3.Stora...
                                             ^^^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”.
  ```
  ...ent method: If you expect to be loading a lot of large files, you may be bet...
                                             ^^^^^^^^
  ```
- Apostrophe might be missing. Did you mean “file’s” (curly possessive apostrophe) or “file's” (straight possessive apostrophe)? (Some terms such as “farmers’ market” have a possessive apostrophe after the first noun.)
  ```
  ...one content archive, and linking to the files inside is much easier if each ...
                                             ^^^^^
  ```
- Possibly missing comma found.
  ```
  ...an array of Files, uploading is simple: IMPORTANT Deleting files from the We...
                                             ^^^^^^^^^
  ```
- Possibly missing article found.
  ```
  ... IMPORTANT Deleting files from the Web3.Storage site's Files page  (opens ne...
                                             ^^^^^^^
  ```
- Consider using a more formal alternative to avoid the overused word “more”.
  ```
  ...to guide on retrieval. You can also get more information about the status of...
                                             ^^^^^^^^^^^^^^^^
  ```

**Review by terichadbourne — APPROVED** [review]

**Pull request merged**